compiler/vm.nim

```diff
@@ -494,7 +494,7 @@ proc rawExecute(c: PCtx, start: int, tos: PStackFrame): TFullReg =
     of opcLdObj:
       # a = b.c
       decodeBC(rkNode)
-      let src = regs[rb].node
+      let src = if regs[rb].kind == rkNodeAddr: regs[rb].nodeAddr[] else: regs[rb].node
```

Comment:
Much better than nothing, but it indicates that the VM codegen generates wrong instruction sequences. Are you sure this is the best patch to fix it? I know, the VM codegen is way harder to fix. ;-)